policyengine/impact/household/charts.py

```diff
@@ -461,8 +478,11 @@ def mtr_chart(
         line_shape="hv",
     )
     # Shade baseline and reformed net income cliffs.
-    shade_cliffs(baseline, config, fig, charts.GRAY)
-    shade_cliffs(reformed, config, fig, charts.BLUE)
+    ymax = df[y_fig].max() * 1.05  # Add a buffer.
```

Comment:
Sounds good, any particular reason for 5% or is just arbitrary?

Reply:
Arbitrary, tends to align with what plotly makes the y axis with respect to the max value IME